app/controllers/locations_controller.rb

```diff
@@ -6,12 +6,14 @@ def new
 
   def create
     @location = Location.new(location_params_without_blank_ips)
+    number_of_ips_added = @location.ips.length
+    pluralize_adress = 'IP address'.pluralize(number_of_ips_added)
```

Comment:
I wouldn't be opposed to just inlining these.
No need for variables.

As it stands we may declare these variables and never use them if the location failed to save.

Reply:
Fair point sir